### PR TITLE
Bintray

### PR DESCRIPTION
**Pull request by dgmltn**

### CODE REVIEW

File: demo/build.gradle [hunk 1]
```diff
@@ -1,4 +1,15 @@
-apply plugin: 'android'
+buildscript {
```
Comment (brennantaylor) [inline demo/build.gradle]: You can just apply this at the root build.gradle file

File: demo/build.gradle [hunk 2]
```diff
@@ -8,6 +19,18 @@ android {
     compileSdkVersion rootProject.ext.compileSdkVersion
     buildToolsVersion rootProject.ext.buildToolsVersion
 
+    defaultConfig {
+        applicationId "com.dgmltn.sharewhere"
```
Comment (brennantaylor) [inline demo/build.gradle]: Already in the AndroidManifest.

File: gradle/wrapper/gradle-wrapper.properties [hunk 3]
```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=http\://services.gradle.org/distributions/gradle-1.10-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.2.1-all.zip
```
Comment (brennantaylor) [inline gradle/wrapper/gradle-wrapper.properties]: 2.4!

**Comment by brennantaylor** [comment]
@dgmltn Done with my comments. I really want to write a gradle plugin that takes care of the android library upload instead of relying on the grab-bag of scripts that exist.

**Comment by dgmltn** [comment]
@brennantaylor that would be great. It does feel a bit weird to use "apply from: [random gradle script hosted on a git repo]" to do that

**Pull request merged**